src/plugins/media/schema.js

```diff
@@ -1,11 +1,27 @@
-// Copyright 2018 OpenStax Poland
+// Copyright 2019 OpenStax Poland
 // Licensed under the MIT license. See LICENSE file in the project root for
 // full license text.
 
+import { Block, Text } from 'slate'
+
 function normalizeMedia(change, error) {
-    const { code: violation, key, node } = error
+    const { code: violation, node, key } = error
 
     switch (violation) {
+    case 'child_min_invalid':
+        console.log('media child_min_invalid', node.toJS())
+        if (!node.nodes.some(n => n.type && n.type === 'media_alt')) {
```

Comment:
You can use `error.index` to determine from where the node is missing. When it's 0 then it must be an `image`, when 1 it must be a `media_alt`.